cmake/node.xcscheme

```diff
@@ -67,7 +67,7 @@
       </EnvironmentVariables>
    </LaunchAction>
    <ProfileAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Debug"
```

Comment:
...and this one. I originally changed this to Debug because Release wouldn't give me symbols, but that may have been fixed by https://github.com/mapbox/mapbox-gl-native/pull/12672